app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -171,4 +169,7 @@ class MainActivity : AppCompatActivity() {
         startActivity(intent)
     }
 
+    companion object{
+        const val cameraPermissionRequestCode = 123
```

Comment:
Sorry but... as I said in my previous comment constants should be uppercase. And this one should also be private I think

Reply:
Done